components/service/glean/src/main/java/mozilla/components/service/glean/scheduler/GleanLifecycleObserver.kt

```diff
@@ -34,7 +34,7 @@ internal class GleanLifecycleObserver : LifecycleObserver {
      *
      * https://developer.android.com/reference/android/app/Activity.html#onResume()
      */
-    @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
+    @OnLifecycleEvent(Lifecycle.Event.ON_START)
```

Comment:
The comment above mentions `ON_RESUME` :)

Reply:
🤦‍♂️ Updated and amended, thanks!